preferences/launchers.js

```diff
@@ -0,0 +1,101 @@
+        'showFavorites', //show-favorites (b)
+
+        'showRunning', //show-running (b)
+        'workspaceIsolation', //isolate-workspaces (b)
+        'showUrgent', //workspace-agnostic-urgent-windows (b)
+        'isolateMonitors', //isolate-monitors (b)
+
+        'showMounts', //show-mounts (b)
+        'showMountsOnlyMounted', //show-mounts-only-mounted (b)
+        'showMountsNetwork', //show-mounts-network (b)
+        'isolateLocations', //isolate-locations (b)
+
+        'showAppsButton', //show-show-apps-button (b)
+        'moveStart', //show-apps-at-top (b)
+        'animateApps', //animate-show-apps (b)
+        'moveToEdge', //show-apps-always-in-the-edge (b)
+
+        'showTrash', //show-trash (b)
+
+        'focusVisable' //scroll-to-focused-application (b)
+    ]
+}, class Launchers extends Adw.PreferencesPage {
+    constructor(settings) {
+        super({});
+
+        settings.bind(
+            'show-favorites', this._showFavorites, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+
+        settings.bind(
+            'show-running', this._showRunning, 'enable-expansion',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+        settings.bind(
+            'isolate-workspaces', this._workspaceIsolation, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+        settings.bind(
+            'isolate-monitors', this._isolateMonitors, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+
+
+        settings.bind(
+            'show-mounts', this._showMounts, 'enable-expansion',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+        settings.bind(
+            'show-mounts-only-mounted', this._showMountsOnlyMounted, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+        settings.bind(
+            'show-mounts-network', this._showMountsNetwork, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+        settings.bind(
+            'isolate-locations', this._isolateLocations, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+
+
+        settings.bind(
+            'show-show-apps-button', this._showAppsButton, 'enable-expansion',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+        settings.bind(
+            'show-apps-at-top', this._moveStart, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+        settings.bind(
+            'animate-show-apps', this._animateApps, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+        settings.bind(
+            'show-apps-always-in-the-edge', this._moveToEdge, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+
+        settings.bind(
+            'show-trash', this._showTrash, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+
+        settings.bind(
+            'scroll-to-focused-application', this._focusVisable, 'active',
+            Gio.SettingsBindFlags.DEFAULT
+        );
+    }
+});
```

Comment:
This looks very formulaic, personally I'd do something more like this:

```suggestion

        bindOptions = [
            ['show-favorites', this._showFavorites, 'active'],
            ['show-running', this._showRunning, 'enable-expansion'],
            ['isolate-workspaces', this._workspaceIsolation, 'active'],
            ['isolate-monitors', this._isolateMonitors, 'active'],
            ['show-mounts', this._showMounts, 'enable-expansion'],
            ['show-mounts-only-mounted', this._showMountsOnlyMounted, 'active'],
            ['show-mounts-network', this._showMountsNetwork, 'active'],
            ['isolate-locations', this._isolateLocations, 'active'],
            ['show-show-apps-button', this._showAppsButton, 'enable-expansion'],
            ['show-apps-at-top', this._moveStart, 'active'],
            ['animate-show-apps', this._animateApps, 'active'],
            ['show-apps-always-in-the-edge', this._moveToEdge, 'active'],
            ['show-trash', this._showTrash, 'active'],
            ['scroll-to-focused-application', this._focusVisable, 'active']
        ];

        bindOptions.forEach((options) => {
            settings.bind(options[0], options[1], options[2],
            Gio.SettingsBindFlags.DEFAULT);
        });
    }
});
```

This would also apply to the other preferences scripts. (This code was written on GitHub, I haven't tested it)

But I'm not a maintainer of this project, so styling would be up to someone like @3v1n0

Reply:
Looks promising, I'll will test it later.
Thanks for the suggestion!